R4ML/R/sysml.bridge.R

```diff
@@ -82,10 +82,10 @@ sysml.MatrixCharacteristics <- setRefClass("sysml.MatrixCharacteristics",
 sysml.MLContext <- setRefClass("sysml.MLContext",
   fields = list(env="environment"),
   methods = list(
-    initialize = function(sparkContext = sysmlSparkContext) {
+    initialize = function(sparkContext = r4ml.env$sysmlSparkContext) {
       env <<- new.env()
       if (missing(sparkContext)) {
-        sparkContext = get("sysmlSparkContext", .GlobalEnv)
+        sparkContext <- r4ml.env$sysmlSparkContext
```

Comment:
That is going to be a pain in the but. It would be easier if we could create a whole new environment instead of having putting it in a variable in the existing environment. Regardless is this still even necessary?